.yamato/_projectcontext.yml

```diff
@@ -332,38 +332,38 @@ test_all_project_macos_CUSTOM-REVISION:
               -  "**/test-results/**"
     variables:
         CUSTOM_REVISION: custom_revision_not_set
-test_all_project_windows_min_editor:
+test_all_project_Win_min_editor:
     name: Test minimum editor version - all packages [project context] Win
     agent:
         type: Unity::VM
         image: package-ci/win10:stable
         flavor: b1.large
     dependencies:
       -  .yamato/_projectcontext.yml#pack_all_project
-      -  .yamato/_editor_priming.yml#editor:priming:test_min_editor:windows
+      -  .yamato/_editor_priming.yml#editor:priming:test_min_editor:Win
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  upm-ci project test -u .\.Editor --project-path TestProjects/SRP_SmokeTest --type vetting-tests --extra-create-project-arg="-upmNoDefaultPackages"
```

Comment:
I think we still need that extra argument since it's a upm-ci test command and it won't be added by default before Feb. 15th

Reply:
missed that one, thanks!